src/frontend/magnify/src/components/meetings/CreateMeetingForm/CreateMeetingForm.tsx

```diff
@@ -53,9 +55,14 @@ const messages = defineMessages({
     defaultMessage: 'Create meeting',
     description: 'Label for the submit button',
   },
+  cancelLabel: {
+    id: 'components.meetings.CreateMeetingForm.cancelLabel',
+    defaultMessage: 'Cancel',
+    description: 'Label for the cancel button',
+  },
 });
 
-const CreateMeetingForm = ({ roomSlug }: CreateMeetingFormProps) => {
+const CreateMeetingForm = ({ roomSlug, onCancel, onSuccess }: CreateMeetingFormProps) => {
```

Comment:
Maybe add a test in CreateMeetingForm.test.tsx to check the `onCancel` and `onSuccess` behavior ?
(If we have enough time left, of course… WDYT @sampaccoud ? )

Reply:
not much time left indeed so best effort :man_shrugging:

Reply:
@kernicPanel I added simple tests for both `onCancel` and `onSuccess`.